index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <nav>
+        <div id="nav-container">
+            <div class="nav-item">
+                <h1>Welcome</h1>
+            </div>
+            <div class="nav-item nav-item2"><img src="img/Icon - Menu.png" alt="Menu Icon"></div>
+        </div>
+    </nav>
+
+    <section>
+        <div id="home">
+            <div class="welcome-text first-welcome-text">
+                <p>Hey there. I’m Raphael
```

Comment:
[ ] this is the main title of your website, so it should be an `h1` tag, we use the `h1` tag If we want that google (and other search engines) know about a specific topic. kindly fix that 🤓.